### DISCUSSION TITLE
Is there a limit withing flux image automation/limit cotroller for parallel scan/update ?

### DISCUSSION THREAD

**Discussion by atulnslhub** [fluxcd/flux2#3750] [Q&A] [answered]
We are using fluxcd along argocd to deploy our microservices, When we migrate the microservice images from one env to another flux is unable to update the git repos , we currently have 90-100 microservices which all update at same time during migration.

**Comment by makkes** [comment]
> flux is unable to update the git repos

This is a very vague statement. Please explain in detail what you are experiencing and what you would expect to happen. Attach any error logs or similar output.

**Comment by kingdonb** [comment, marked as answer]
We cannot do anything with this report, please observe the community standards document for guidance about how to better structure your support inquiry. Noise on this community forum is against the ideal of best-effort and community.

https://fluxcd.io/support/#getting-help-from-the-flux-community
